src/LLVM_Runtime_Linker.cpp

```diff
@@ -830,7 +830,7 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
                 modules.push_back(get_initmod_x86_cpu_features(c, bits_64, debug));
             }
             if (t.arch == Target::ARM) {
-                if (t.bits == 64) {
+                if (t.bits != 64) {
```

Comment:
I know this is a nit, but can we use `t.bits == 64` and flop the clauses instead, since that's the pattern used everywhere else in this file?

Reply:
done